hooks/useMulticallWrite/useMulticallWrite.ts

```diff
@@ -14,8 +16,19 @@ export function useMulticallWrite(
     functionName: 'multicall',
     args: [calldata as `0x${string}`[]],
   });
+  const configWithGasOverride = useMemo(() => {
+    if (!multicallConfig.request) return multicallConfig;
```

Comment:
In what cases is this not set? If it starts out not set, how long does it take to propagate? Should we do anything to prevent starting a write until we can pad the gas?

Reply:
we need to add some extensive booleans to the `skip` param and then this will always be set. I say we do it when i rework the borrow page